pkg/asset/installconfig/clusterid.go

```diff
@@ -74,6 +75,15 @@ func generateInfraID(base string, maxLen int) string {
 	}
 	base = strings.TrimRight(base, "-")
 
+	// Return the InfraID provided by the "INFRA_ID" environment variable if valid
+	if envInfraID, ok := os.LookupEnv("INFRA_ID"); ok && envInfraID != "" {
```

Comment:
- Can you change to "OPENSHIFT_INSTALL_INFRA_ID_OVERRIDE" to comply with the rest of the env vars
- Please change to:
if envInfraID, ok := os.LookupEnv

and add a warn log message like in:
https://github.com/openshift/installer/blob/6d778f911e79afad8ba2ff4301eda5b5cf4d8e9e/pkg/asset/rhcos/image.go#L51-L55

Reply:
Sure